app/controllers/campaigns_controller.rb

```diff
@@ -128,6 +128,7 @@ def campaign_json(campaign: @campaign)
     ret['amount_raised'] = campaign.amount_raised
     ret['amount_allocated'] = campaign.amount_allocated
     ret['last_contribution'] = campaign.last_contribution
+    ret['seller_id'] = campaign.seller.nil? ? "" : campaign.seller.seller_id
```

Comment:
I think we want to update the frontend to use `seller_distributor_pairs` instead

Reply:
Yes because there could be a project vs a seller id (normal gam vs mega gam) in the frontend- are you saying we should just keep the backend as it is?

Reply:
Right I've updated the regular campaigns to return the seller/dist that's related to the campaign directly as a seller_distributor_pair since we're going to be removing campaign.seller and campaign.dist in favor of seller_dist_pairs

Reply:
i had created a ticket for this if someone wanted to take it up:
https://trello.com/c/wMetd5qi/708-megagam-migrate-campaign-seller-distributor-fields-to-use-the-campaignssellersdistributors-join-table

Reply:
ticket is to migrate seller_id and distributor_id to the seller/distributor table and remove old code